CHANGELOG.md

```diff
@@ -1,3 +1,118 @@
+# 1.9.0-beta01 (July 2025)
+
+_Changes since 1.9.0-alpha03_
+
+## Highlights
+
+### iOS
+
+- Remove experimental attribute from `UIKitInteropInteractionMode` [#2215](https://github.com/JetBrains/compose-multiplatform-core/pull/2215)
```

Comment:
Doesn't look like a Highlight (Major feature). Move it to Migrations notes. Change the PR itself, and mention this in the PR comment